bool control_mode
---
bool result

